NEWS.md

```diff
@@ -0,0 +1,302 @@
+# 1.7
+
+<!--
+TODO:
+- reword things
+- thank more ppl?
+- sub/3 fixes?
+- validate module metadata is object fix?
+- fix number lexer to avoid conflict with object indexing (#2254)
+- mention some build system changes?
+-->
+
+Lots of JSON has been queried and transformed since the stable release 5 years ago, so now we're happy to finally do a new stable release!
```

Comment:
```suggestion
Lots of JSON has been queried and transformed since the last stable release, jq-1.6, 5 years ago, so now we're happy to finally do a new stable release!
```